Yosemite/Yosemite/SnapshotsProvider/FetchResultSnapshotsProvider.swift

```diff
@@ -183,6 +191,47 @@ public final class FetchResultSnapshotsProvider<MutableType: FetchResultSnapshot
     }
 }
 
+// MARK: - FetchedResultsController Activation
+
+@available(iOS 13.0, *)
+private extension FetchResultSnapshotsProvider {
+    /// Start `fetchedResultsController` fetching and dispatching of snapshots.
+    func activateFetchedResultsController() throws {
+        try fetchedResultsController.performFetch()
+    }
+
+    /// Returns `true` if the `activateFetchedResultsController()` method was previously called.
+    var fetchedResultsControllerIsActive: Bool {
+        fetchedResultsController.fetchedObjects != nil
```

Comment:
it's interesting that this doesn't crash the app when accessing `fetchedObjects` after the reset and before `performFetch()` again 🤔 like in the [stack trace](https://sentry.io/organizations/a8c/issues/1703307167/?project=1458804&query=Object%27s+persistent+store+is+not+reachable&statsPeriod=14d), it's crashing from getting the `fetchedObjects` from product FRC. I might just misunderstand the root cause of this crash 😅 

update: I tested adding `print(fetchedResultsController.fetchedObjects)` to above this nil check, and the app crashes on this print. It's possible that doing a nil check is fine while accessing the individual fetched objects is not (nil check might not involve accessing the objects from the store).